source/css/main.css.scss

```diff
@@ -0,0 +1,5 @@
+@charset 'utf-8';
+
+@import
+  '_normalize.css'
```

Comment:
Line should be indented 0 spaces, but was indented 2 spaces<br>Prefer double-quoted strings<br>Declaration should not have a space before the terminating semicolon